std/conv.d

```diff
@@ -177,15 +166,17 @@ $(I UnsignedInteger):
  */
 template to(T)
 {
+    ///
     T to(A...)(A args)
-        if (!isRawStaticArray!A)
+        if (A.length > 0 && !isStaticArray!(A[0]))
```

Comment:
Ideally we should have 2 overloads instead of just using `args` here, I can do that after this pull is merged.